server/sonar-web/src/main/js/apps/projectActivity/components/GraphsLegendItem.js

```diff
@@ -32,11 +33,12 @@ type Props = {
   style: string,
   removeMetric?: string => void
 };
+*/
 
 export default class GraphsLegendItem extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
-  handleClick = (e: Event) => {
+  handleClick = (e /*: Event */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)